spec/classes/chrony_spec.rb

```diff
@@ -40,6 +40,27 @@
             it { is_expected.to contain_file('/etc/chrony.keys').with_replace(true) }
             it { is_expected.to contain_file('/etc/chrony.keys').with_content("0 xyzzy\n") }
           end
+        when 'Gentoo'
+          context 'using defaults' do
+            it do
+              is_expected.to contain_file('/etc/chrony/chrony.conf')
+                .without_content(%r{^\s*cmdallow})
+                .with_content(%r{^\s*server 0.pool.ntp.org iburst$})
+                .with_content(%r{^\s*server 1.pool.ntp.org iburst$})
+                .with_content(%r{^\s*server 2.pool.ntp.org iburst$})
+                .with_content(%r{^\s*server 3.pool.ntp.org iburst$})
+                .with_content(%r{^\s*rtconutc$})
+                .with_content(%r{^\s*driftfile /var/lib/chrony/drift$})
+                .with_content(%r{^\s*rtcsync$})
+                .without_content(%r{^\s*dumpdir})
+              is_expected.to contain_file('/etc/chrony/chrony.keys')
```

Comment:
I would still keep this in a separate `it {}` block so you only have one `is_expected.to` per `it {}`.